tests/src/test/java/edu/snu/onyx/tests/runtime/master/scheduler/FaultToleranceTest.java

```diff
@@ -181,11 +168,42 @@ public void testContainerRemoval() throws Exception {
     final IREdge e4 = new IREdge(DataCommunicationPatternProperty.Value.OneToOne, v4, v5, Coder.DUMMY_CODER);
     irDAGBuilder.connectVertices(e4);
 
-    final DAG<IRVertex, IREdge> irDAG = CompiletimeOptimizer.optimize(irDAGBuilder.buildWithoutSourceSinkCheck(),
-        new TestPolicy(), "");
-    final DAG<PhysicalStage, PhysicalStageEdge> physicalDAG = irDAG.convert(physicalPlanGenerator);
+    DAG<IRVertex, IREdge> irDAG;
+    DAG<PhysicalStage, PhysicalStageEdge> physicalDAG = null;
+    try {
```

Comment:
This can be removed and `physicalDAG` can be assigned to a non-null value,  if `buildPlan()` throws an Exception.